.travis.yml

```diff
@@ -23,8 +23,6 @@ env:
     - TF_VERSION_ID=  # Do not install TensorFlow in this case
 
 cache:
-  # Reuse the pip cache directory across build machines.
-  pip: true
```

Comment:
`pip: false` then explain why opted out? Some stupid person like me may flip it back on thinking that it will improve our CI :)

Reply:
Good idea; thanks. Done.